exchange/exchange_test.go

```diff
@@ -3443,6 +3443,41 @@ func TestUpdateHbPbCatDur(t *testing.T) {
 	}
 }
 
+func TestInsertStoredImpData(t *testing.T) {
+	impsToStoredRequest := make(map[string]ImpExtInfo)
```

Comment:
Please rename this to `impExtInfoMap` for consistency with the new names

Reply:
yes, renamed